desktop-styles.css

```diff
@@ -0,0 +1,476 @@
+@media screen and (min-width: 48.06144rem) {
+  html {
+    font-size: 62.5%;
+  }
+
+  body {
+    max-width: 144rem;
+    max-height: auto;
+    font-size: 1.6rem;
+  }
+
+  #top-menu {
+    display: flex;
+  }
+
+  #top-menu ul {
+    list-style-type: none;
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  #top-menu a {
+    text-decoration: none;
+    width: 6.3rem;
+    height: 2rem;
+    font-weight: 500;
+    font-size: 1.5rem;
+    line-height: 2rem;
+    text-align: center;
+    color: #344563;
+  }
+
+  .logo {
+    padding: 0 15.4rem;
+  }
+
+  .logo button {
+    display: none;
+  }
+
+  #landing-page {
+    min-width: 144rem;
+    min-height: 91.6rem;
+  }
+
+  .main-info {
+    max-width: 139.74rem;
+    max-height: 72.146rem;
+    background-image: url("./images/dt-bg.jpg");
+    background-size: cover;
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+    margin: 6.3rem  0.8rem 7.654rem 3.46rem;
+    padding: 0;
+    display: flex;
+    align-items: center;
+  }
+
+  .main-info h2 {
+    width: 92rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    padding-top: 22.1rem;
+  }
+
+  #break1 {
+    display: none;
+  }
+
+  .info-break {
+    display: block;
+  }
+
+  .my-info {
+    width: 92rem;
+    max-height: 9.8rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: wrap;
+    margin-bottom: 5.5rem;
+    margin-left: 0;
+  }
+
+  .contact-info {
+    width: 92rem;
+    height: 5.8rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: flex-start;
+    flex-wrap: nowrap;
+    margin-bottom: 24rem;
+  }
+
+  .project-list {
+    display: grid;
+    max-height: 269.4rem;
+    min-width: 144rem;
+    grid: 1fr;
+    align-items: center;
+    justify-content: space-evenly;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 14.2rem;
+  }
```

Comment:
- [ ] Project listing is broken on mobile with text going out of boundaries and horizontal scrolling
![image](https://user-images.githubusercontent.com/33767951/172972579-8845af98-9648-4ffa-956a-45d6e4fa11f8.png)